src/frontend/src/components/FilterPanel/index.tsx

```diff
@@ -212,7 +214,7 @@ const FilterPanel: FC<Props> = ({
   };
 
   return (
-    <StyledFilterPanel>
+    <StyledFilterPanel isOpen={isOpen}>
```

Comment:
Doing this via the `display: ${isOpen ? "block" : "none"};` is totally reasonable, but I have a slight preference for instead just making the return short-circuit to `null` for the entire FilterPanel when `isOpen` is false. It's probably six-in-one, half-dozen in the other, but that way if there are any unexpected side-effects in a downstream child, they can't fire off while still being `display: none` hidden. That said, I didn't fully trace the filtering process while I was working on it: if those child components not "rendering" in a hidden way will drop the filter state, I think the display none approach is definitely the right way.

Reply:
I also actually have a preference for that, but then we just push the problem of state disappearing down a level. If I return `null` instead of hiding the panel, we lose the chip state even though the samples themselves stay filtered.

Reply:
Whoops, yeah. I had forgotten about the state in those lower levels, yup, definitely wouldn't work.